pages/Fisherman-Androids.md

```diff
@@ -0,0 +1,39 @@
+An Android that specializes in automating the fishing process.
+
+## Obtaining
+Fisherman Androids can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+You have to put water underneath the Android in order for it to work.
+Androids need fuel in order to work, you have to fill them with the [Required Fuel](https://github.com/Slimefun/Slimefun4/wiki/Normal-Androids#power-source) before booting!
+
+## Tiers
+There are three tiers on the Fisherman Android, each with different modifiers listed below.
+
+| Tier      | Fuel Efficiency | Success Rate |
+| --------- | :-------------: | ------------ |
+| Basic     | 1.0x            | 10%          |
+| Advanced  | 1.5x            | 20%          |
+| Empowered | 8.0x            | 30%          |
+
+## Loot Table
+These are items that you can get when using the Fisherman Android:
+| Item           |
+| -------------- |
+| Raw Cod        |
+| Raw Salmon     |
+| Tropical Fish  |
+| Pufferfish     |
+| Cooked Cod     |
+| Cooked Salmon  |
+| Bone           |
+| String         |
+| Ink Sac        |
+| Kelp           |
+| Stick          |
+| Rotten Flesh   |
+| Leather        |
+| Bamboo         |
+| Saddle         |
+| Name Tag       |
+| Nautilus Shell |
```

Comment:
A one-column table is a bit weird I think. Perhaps a list would be better here.

Reply:
changed to a list